core/string.cpp

```diff
@@ -23,13 +23,13 @@ std::ostream &operator<<(std::ostream &os, const Dimensions &dims) {
 
 std::string to_string(const Dimensions &dims) {
   if (dims.empty())
-    return "{}";
-  std::string s = "{{";
+    return "()";
+  std::string s = "(";
   for (int32_t i = 0; i < scipp::size(dims.shape()); ++i)
-    s += to_string(dims.labels()[i]) + ", " + std::to_string(dims.shape()[i]) +
-         "}, {";
-  s.resize(s.size() - 3);
-  s += "}";
```

Comment:
Why round parens? Curly braces seem more natural with the colon notation in Python.

Reply:
Because that is what we use in the HTML repr.

Reply:
Ok, I think I would prefer braces in both places but it is not really relevant because there is no place where we can specify dimensions as a dict. (Right?)

Reply:
Right. I think we use parenthesis because xarray does.